lang/fr/texts/selected-media.html

```diff
@@ -0,0 +1,23 @@
+<div class="row text-center">
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/lemonde.png" alt="" style="height:145px">
+		<h3>2013 - Le Monde</h3>
+		<p> 
+			<a href="http://www.lemonde.fr/sante/article/2013/04/15/alimentation-face-aux-doutes-les-internautes-s-organisent_3159792_1651302.html">Alimentation: face aux doutes, les internautes s'organisent.</a> Interview croisée d'Open Food Facts et Notéo dans Le Monde.
+		</p>
+	</div>
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/media-france2-envoyespecial.jpeg" alt="" style="height:145px">
+		<h3>2018 - Envoyé Spécial sur les aliments ultra-transformés</h3>
+		<p>Nous avons été mis en valeur dans Envoyé Spécial une deuxième fois en 2018, dans le cadre d'une soirée consacrée aux aliments ultra-transformés. <a href="https://www.youtube.com/watch?v=Tu9RA82-AwQ">La journaliste d'investigation et animatrice Elise Lucet interviewe Stéphane.</a> Envoyé Special est l'émission numéro 1 d'investigation en France, similaire à BBC Panorama.
+		</p>
+	</div>
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/global/global-rfi.jpeg" alt="Une base de données de produits alimentaires" style="height:145px">
+		<h3>2019 - C'est En France - La révolution du scan</h3>
+		<p>Nous avons présenté Open Food Facts sur France 24 dans le cadre d’ <a href="https://youtu.be/GMBQGG2FjP8?t=440">un reportage sur la révolution alimentaire que nous avons lancée en France</a> ( <a href="https://youtu.be/yDZKs_ibJns?t=441">version française</a> ). 
+		</p>
+	</div>
+</div>
+
+<a class="button" href="/presskit_fr_FR_selection">Toutes nos apparitions selectionnées</a>
```

Comment:
That URL doesn't sound like a good idea to me. We shouldn't pollute the root URI with that many different styles, and [Cool URIs don't change](https://www.w3.org/Provider/Style/URI). Wouldn't it be better to host it somewhere else (outside of the actual web app)?